salesforce-gui/src/main/java/org/fundacionjala/salesforce/ui/entities/PersonalInformation.java

```diff
@@ -0,0 +1,65 @@
+package org.fundacionjala.salesforce.ui.entities;
+
+/**
+ * [SL] Entity that represents a Personal Information in Salesforce.
+ */
+public class PersonalInformation {
```

Comment:
This entity is already represented by User entity, because the personal information edited is about the loged id User.

Reply:
I think you should work with "User" entity, please remove this entity.